<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HarishGarg" name="login test cases" tests="12" failures="2" timestamp="21 Oct 2019 09:39:38 GMT" time="0.177" errors="0">
  <testcase name="Login_UI" time="0.047" classname="com.Login">
    <skipped/>
  </testcase> <!-- Login_UI -->
  <testcase name="Login_UI" time="0.004" classname="com.Login">
    <skipped/>
  </testcase> <!-- Login_UI -->
  <testcase name="Login_UI" time="0.0" classname="com.Login">
    <skipped/>
  </testcase> <!-- Login_UI -->
  <testcase name="Login_UI" time="0.0" classname="com.Login">
    <skipped/>
  </testcase> <!-- Login_UI -->
  <testcase name="Login_UI" time="0.0" classname="com.Login">
    <failure type="java.lang.AssertionError" message="expected [fail] but found [Login_UI]">
      <![CDATA[java.lang.AssertionError: expected [fail] but found [Login_UI]
	at com.Login.Login_UI(Login.java:28)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Login_UI -->
  <testcase name="login_google" time="0.0" classname="com.Login">
    <skipped/>
  </testcase> <!-- login_google -->
  <testcase name="login_google" time="0.0" classname="com.Login">
    <skipped/>
  </testcase> <!-- login_google -->
  <testcase name="login_google" time="0.015" classname="com.Login">
    <skipped/>
  </testcase> <!-- login_google -->
  <testcase name="login_google" time="0.0" classname="com.Login">
    <skipped/>
  </testcase> <!-- login_google -->
  <testcase name="login_google" time="0.0" classname="com.Login">
    <failure type="java.lang.AssertionError" message="expected [fail] but found [login_google]">
      <![CDATA[java.lang.AssertionError: expected [fail] but found [login_google]
	at com.Login.login_google(Login.java:15)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- login_google -->
  <testcase name="Signup_UI" time="0.0" classname="com.Signup"/>
  <testcase name="Signup_google" time="0.0" classname="com.Signup"/>
</testsuite> <!-- login test cases -->
